### ISSUE TITLE
Error message "fatal: not a git repository" when using the GIT snapshot

### ISSUE THREAD

**Issue by M-Reimer** [esp8266/Arduino#7744]
### Basic Infos

- [x] This issue complies with the [issue POLICY doc](https://github.com/esp8266/Arduino/blob/master/POLICY.md).
- [x] I have read the documentation at [readthedocs](https://arduino-esp8266.readthedocs.io/en/latest) and the issue is not addressed there.
- [x] I have tested that the issue is present in current master branch (aka latest git).
- [x] I have searched the issue tracker for a similar issue.
- [x] If there is a stack dump, I have decoded it.
- [x] I have filled out all fields below.

#### Platform

- Hardware:				other AI-Thinker ESP8266MOD
- Core Version:			47e7b2d
- Development Env:		Arduino IDE
- Operating System:		Arch Linux

### Settings in IDE

- Module:  			Wemos D1 mini r2
- Flash Mode:			?
- Flash Size: 		4MB/1MB
- lwip Variant:   v2 Lower Memory
- Reset Method:		?
- Flash Frequency:	40Mhz
- CPU Frequency:		80Mhz
- Upload Using:		SERIAL
- Upload Speed:   921600

### Problem Description

If I create a toolchain based on the GIT snapshot (ZIP file), then I always get the following message prior compiling:

    fatal: not a git repository (or any of the parent directories): .git

Reason is that this script tries to fetch some info using GIT:

https://github.com/esp8266/Arduino/blob/master/tools/makecorever.py

The message does not cause any problems. It just looks ugly.

How do you plan to fix this? Hide the message? I could create a pull request if I know how you want this to be fixed.

**Comment by d-a-v** [comment]
Can you try with:
```
diff --git a/tools/makecorever.py b/tools/makecorever.py
index e7bd3486..659591b5 100755
--- a/tools/makecorever.py
+++ b/tools/makecorever.py
@@ -26,7 +26,7 @@ def generate(path, platform_path, git_ver="ffffffff", git_desc="unspecified"):
     def git(*args):
         cmd = ["git", "-C", platform_path]
         cmd.extend(args)
-        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, universal_newlines=True)
+        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, universal_newlines=True, stderr=DEVNULL)
         return proc.stdout.readlines()[0].strip()
 
     try:
```

If that suits your need, would you make a pull request ?

**Comment by M-Reimer** [comment]
Done: https://github.com/esp8266/Arduino/pull/7745

**Comment by d-a-v** [comment]
closed by #7745
